cdap-explore/src/main/java/co/cask/cdap/hive/context/ContextManager.java

```diff
@@ -127,6 +127,7 @@ private static Context createContext(Configuration conf) throws IOException {
       new LocationRuntimeModule().getDistributedModules(),
       new DiscoveryRuntimeModule().getDistributedModules(),
       new DataFabricModules().getDistributedModules(),
+      // here
```

Comment:
Ah, I need to validate whether the bindings there are appropriate.
Will do that.